src/xenia/kernel/xnotifylistener.h

```diff
@@ -30,13 +30,8 @@ union XNotificationKey {
   };
   XNotificationID id;
 
-  static constexpr XNotificationID get_id(uint8_t mask_index,
-                                          uint16_t local_id) {
-    XNotificationKey key = {};
-    key.mask_index = mask_index;
-    key.local_id = local_id;
-    return key.id;
-  }
+  constexpr XNotificationKey(XNotificationID notification_id)
```

Comment:
Additionally, `constexpr operator XNotificationID() const { return id; }` may be nice to have, so the two types can be used interchangeably, for example, while passing them to functions.

Or maybe we should just drop `XNotificationID` at all, and have `constexpr` IDs build via the `uint32_t` constructors? Since the two types mean pretty much the same thing. But this needs bigger changes, such as adding `operator==`, `operator!=`.